src/Common/src/Interop/Windows/Interop.Libraries.cs

```diff
@@ -5,9 +5,13 @@ internal static partial class Interop
 {
     private static class Libraries
     {
+        internal const string Console_L1 = "api-ms-win-core-console-l1-1-0.dll";
+        internal const string Console_L2 = "api-ms-win-core-console-l2-1-0.dll";
+        internal const string CoreFile = "api-ms-win-core-file-l1-1-0.dll";
```

Comment:
File is a little different , as I add more ApiSets here, we will see some non-core file Apisets as well :).